lib/tasks/superhero_api.rake

```diff
@@ -0,0 +1,90 @@
+namespace :superhero do
+  desc "Import data from external Superhero API"
+
+  task import_from_api: :environment do
+    require 'faraday'
+    require 'faraday_middleware'
+    require 'json'
+
+    id = 1
+
+    conn = Faraday.new(url: "https://superheroapi.com/api/10111528630893278/") do |faraday|
+      faraday.request :url_encoded
+      faraday.response :json, content_type: /\bjson$/
+      faraday.use FaradayMiddleware::FollowRedirects, limit: 5
+    end
+    
+
+    loop do
+        response = conn.get("#{id}")
+
+        if response.body['response'] == "success"
+          data = response.body
+
+          character = Character.find_or_initialize_by(id: data['id'])
```

Comment:
This looks great.  We can look at potentially coming to revisit for when we need to update our DB.  if we decide to update x amount a times a month.